src/welcome.ui

```diff
@@ -107,9 +122,21 @@
                     <property name="visible">True</property>
                     <property
                       name="label"
-                    >&lt;big&gt;&lt;b&gt;Mercurial&lt;/b&gt;&lt;/big&gt;
-
-    To set Commit as default editor for Mercurial, set the following in your &lt;tt&gt;hgrc&lt;/tt&gt;</property>
+                    >&lt;big&gt;&lt;b&gt;Mercurial&lt;/b&gt;&lt;/big&gt;</property>
+                    <property name="use-markup">True</property>
+                    <property name="wrap">True</property>
+                    <property name="halign">GTK_ALIGN_START</property>
+                    <property name="xalign">0</property>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkLabel">
+                    <property name="visible">True</property>
+                    <!-- TRANSLATORS: do not replace '<tt>hgrc</tt>' -->
```

Comment:
@pervoj is this easily understand by translators? is there a better way?

Reply:
I don't think it's hard to understand, but I don't know how different translation editing applications would deal with the XML tags. But I don't think it should be a problem.